spacy_transformers/layers/transformer_model.py

```diff
@@ -147,6 +147,8 @@ def _convert_transformer_inputs(model, tokens: BatchEncoding, is_train):
 
 def _convert_transformer_outputs(model, inputs_outputs, is_train):
     _, tensors = inputs_outputs
+    if hasattr(tensors, "to_tuple"):
+        tensors = tensors.to_tuple()
```

Comment:
Okay so this just converts the outer container right? It doesn't touch the actual tensors? Seems fine.

Reply:
This seemed to be the most minimal way to detect `ModelOutput` and convert it to the tuple format where the current methods that use the shape to detect elements work.

Longer term you'd want to support both, or switch everything `ModelOutput` once you drop earlier versions. (Exactly which models and which versions is not clear to me at this point.)